techniques/aria/ARIA11.html

```diff
@@ -32,7 +32,7 @@ <h1>Using ARIA landmarks to identify regions of a page</h1>
          <li><code class="language-html">contentinfo</code>: A region that contains information about the parent document such as copyrights and links to privacy statements.</li>
       </ul>
 
-      <p>There are cases when a particular landmark role could be used more than once on a page, such as on primary and secondary blocks of navigation. In these cases, identical roles should be labeled using a valid technique for labelling regions.</p>
+      <p>There are cases when a particular landmark role could be used more than once on a page, such as on primary and secondary blocks of navigation. In these cases, identical roles should be named using a valid technique for labeling regions.</p>
```

Comment:
Note that I meant to put this in as a suggestion, so flagging that it is commited. If agreement, it can stand.